test/integration_tests/wav2vec2_pipeline_test.py

```diff
@@ -61,7 +61,6 @@ def test_pretraining_models(bundle):
 )
 def test_finetune_asr_model(
         bundle,
-        lang,
```

Comment:
do we also need to remove the corresponding column in the parameters above?

Reply:
We cannot remove the one above. PyTest's `fixture` mechanism is very different from what a normal Python code would behave. Here is the gist of what happens according to my understanding.
1. When PyTest purses test code, it inspects the function argument
2. If the argument name matches a pre-defined fixture, it will be substituted with predefined fixture.
3. If an argument name matches with an argument name of a known fixture, then PyTest will bind it.

So, what happens here is that, PyTest runner, at import time, notices that `lang` is defined, and there is a fixture called `sample_speech(lang, ...)`, which takes `lang` as argument, so it automatically calls `smple_speech` function with `lang` parameter and give the resulting object to test function.
As I have not really used PyTest, this was very confusing behavior, and which is why I forgot to clean up the code and ended up leaving the `lang` argument there.